pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -297,6 +297,35 @@ func (m *podContainerManagerImpl) GetAllPodsFromCgroups() (map[types.UID]CgroupN
 	return foundPods, nil
 }
 
+func (m *podContainerManagerImpl) DoNotEnforceCpuLimits() {
+	m.enforceCPULimits = false
+}
+
+func (m *podContainerManagerImpl) RemoveCpuLimits(pod *v1.Pod) {
+	if !m.enforceCPULimits {
+		klog.InfoS("cpu quota for pod already disabled", "podName", pod.Name, "podUID", pod.UID)
+		return
+	}
+
+	klog.InfoS("Removing cpu quota for pod", "podName", pod.Name, "podUID", pod.UID)
+
+	// Make sure the limits will stay disabled
+	m.enforceCPULimits = false
```

Comment:
when this can be turned to `true` again?

Reply:
No, if I am reading the code correctly, PodContainerManager is not persisted and is created anew every time it is needed in SyncPod.